Allow focus of pdf editor by passing args for using keybindings

excetara2 opened the pull request:
Fixed and updated issue as discussed in using r-stein solution #680

r-stein commented:
I think the change should be done on [line `116`](https://github.com/excetara2/LaTeXTools/blob/master/jumpToPDF.py#L116) and `forward_sync` could also be passed as an argument. And it would be nice to write something in the [README](https://github.com/excetara2/LaTeXTools#forward-and-inverse-search), so people have the chance to know it without reading the python code.

*Side remark: If you push on your branch, the commits will be included in the pull request. Hence you don't have to create a new one if you made changes. In addition I would recommend to not use your masterbranch for future PRs.*

excetara2 commented:
Okay I've updated it so both arguments can be passed and updated the README to state the toggles can be overwritten by the passed in arguments.

_Any future pull requests I'll make a new branch_

ig0774 commented:
Thanks! I've integrated this fix into v3.7.10.
